### ISSUE TITLE
Define DefaultDisplayPropertySet for types

### ISSUE THREAD

**Issue by BrianTJackett** [microsoftgraph/msgraph-sdk-powershell#35]
Suggestion to add DefaultDisplayPropertySet for objects returned from Microsoft Graph.  Currently tested commands have all properties displayed in table format, many of which are not populated with any information.  This requires use of "| select-object -property propA, propB, etc." (or "| format-list ...") to display the information desired.

Example commands tested: Get-Group, Get-User.

For reference implementation:
[PowerShell - Defining Default Member Sets for Objects](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/defining-default-member-sets-for-objects?view=powershell-6)

[PowerShell - Extending properties for obejcts - Property Sets](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/extending-properties-for-objects?view=powershell-6#property-sets)

**Comment by darrelmiller** [comment]
The AutoREST generator provides hooks to help generate these.  We will probably create some samples of how it is done and then start reaching out to teams to get them to help us select the sane defaults.

**Comment by peombwa** [comment]
This has been resolved in version 0.1.4.
``` ps
Install-Module Microsoft.Graph.Beta -RequiredVersion 0.1.4 -Repository GraphPowerShell -Force -AllowClobber
```
